app/models/user.rb

```diff
@@ -359,8 +359,14 @@ def reindex_referencing_models
     end
   end
 
+  def locale
+    (super || AlaveteliLocalization.locale).to_s
+  end
+
   def get_locale
-    (locale || AlaveteliLocalization.locale).to_s
+    warn %q([DEPRECATION] User#get_locale will be removed in 0.37.
```

Comment:
Actually this needs to be `0.38` – `0.37` will introduce the deprecation. Will fixup.